backend/program/media/item.py

```diff
@@ -168,13 +159,14 @@ def set(self, key, value):
 class Movie(MediaItem):
     """Movie class"""
 
-    def __init__(self, item):
+    def __init__(self, item, parent_item_id: Optional[ItemId] = None):
```

Comment:
a movie will never have a parent_item, is this just to avoid logic elsewhere?

Reply:
No this was just me being sloppy.  Good catch.